free-courses-en.md

```diff
@@ -246,8 +246,12 @@
 #### React
 
 * [Start Using React to Build Web Applications](https://egghead.io/courses/react-fundamentals)
+* [The Road to learn React.js](https://roadtoreact.com/)
```

Comment:
maybe the URL should be https://roadtoreact.com/course-details?courseId=THE_ROAD_TO_LEARN_REACT